distribution/smarthome/conf/things/demo.things

```diff
@@ -1,3 +1,3 @@
-yahooweather:weather:berlin [ location="638242" ]
-yahooweather:weather:losangeles "Los Angeles" [ location="2442047", unit="us", refresh=120 ]
+yahooweather:weather:berlin [ location=638242 ]
+yahooweather:weather:losangeles "Los Angeles" [ location=2442047, unit="us", refresh=120 ]
```

Comment:
Normally I would prefer to use separate commits  for different stuff (so a cleanup should not be mixed with a new feature). The different commits could be (sometime) merged by the same PR, because a PR themselves is not visible in the Git history. It could just confuse people if they are looking at the history of demo things to identify why the quotes has been removed and the commit is "Changed signature of operation getConfigStatus...".

But perhaps we should not be to strict here. :wink: